### PR TITLE
Fix monitor run modes

### PR DESCRIPTION
**Pull request by dkijania**
Fix monitor runs for vitup and hersir. Remove unnecessary printout and debug info

### CODE REVIEW

File: src/jormungandr/explorer/src/db/indexing.rs [hunk 1]
```diff
@@ -170,28 +170,6 @@ impl ExplorerBlock {
...
-                        let tx = tx.as_slice();
-                        Some(ExplorerTransaction::from(
-                            &context,
-                            &fragment_id,
-                            &tx,
-                            Some(Certificate::UpdateVote(tx.payload().into_payload())),
-                            offset,
-                            &current_block_txs,
-                        ))
-                    }
```
Comment (kukkok3) [inline src/jormungandr/explorer/src/db/indexing.rs:194]: this needs to be reverted

File: src/jormungandr/testing/mjolnir/src/mjolnir_lib/generators/fragment_generator.rs [hunk 2]
```diff
@@ -259,11 +259,13 @@ impl<'a, S: SyncNode + Send> FragmentGenerator<'a, S> {
...
                 let stake_pool = self.active_stake_pools.get(index).unwrap();
+
                 self.fragment_sender
                     .send_owner_delegation(&mut self.sender, stake_pool, &self.node)
             }
             4 => {
                 let stake_pool = StakePool::new(&self.sender);
+                self.active_stake_pools.push(stake_pool.clone());
                 self.fragment_sender.send_pool_registration(
                     &mut self.sender,
```
Comment (kukkok3) [inline src/jormungandr/testing/mjolnir/src/mjolnir_lib/generators/fragment_generator.rs:270]: this needs to be reverted

**Review by kukkok3 — CHANGES_REQUESTED** [review]
A couple of files need to be reverted

File: src/jormungandr/testing/mjolnir/src/mjolnir_lib/generators/fragment_generator.rs (hunk 2 above)
Comment (dkijania) [reply src/jormungandr/testing/mjolnir/src/mjolnir_lib/generators/fragment_generator.rs]: done

File: src/jormungandr/explorer/src/db/indexing.rs [hunk 3]
```diff
@@ -170,28 +170,6 @@ impl ExplorerBlock {
...
                     Fragment::Transaction(tx) => {
```
[the remark this answers is not in the document]
Comment (dkijania) [reply src/jormungandr/explorer/src/db/indexing.rs:195]: hmm, i think it is reverted, see latest state of branch:

https://github.com/input-output-hk/catalyst-core/blob/fix_hersir_run_modes/src/jormungandr/explorer/src/db/indexing.rs#L173

**Review by kukkok3 — APPROVED** [review]

**Pull request merged**